### PR TITLE
add welcome page and link updates

### PR DESCRIPTION
**Pull request by spencerhandley**
This add a welcome page using this strategy 
https://premium.wpmudev.org/blog/plugin-welcome-screen/

It also includes some other small fixes and an updated read me file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
+Patreon Wordpress is a free plugin built and maintained by Patreon. When a patron pledges, they can then head over to www.yourwordpresssite.com and click your “Connect with Patreon” button to let the site know they are a patron. When you create content on WordPress you will be given the option to limit it to only paying patrons.
```
Comment (telaviv) [inline README.md]: Needs a title

File: README.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+### Installing the Patreon WordPress Plugin
+
+1. Install the plugin
+3. Activate the plugin and then click on ‘Patreon Settings’ to view the options page. It will look something like the below, copy the ‘Redirect URI‘ into the clip board and keep for the next step.
+![](https://c5.patreon.com/external/platform/wordpress-install-ss1.jpeg)
+
+### Creating an oAuth client on Patreon
+
+1. Ensure you are logged into Patreon, using your creator account. When it comes to testing this out you will want to have two Patreon accounts, one that acts as a patron and one being the content creator. The oAuth client needs to be created on your content creating account.
```
Comment (telaviv) [inline README.md]: `Ensure you are logged into Patreon, using your creator account.` -> remove comma

File: README.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+2. Visit the [oAuth client page here](https://www.patreon.com/portal/registration/register-clients) and click "Create Client", fill out the form and add in an image URL for the icon that will appear to users when they are connecting to your website.
+3. Note the field for ‘Redirect URI‘, this should still be in your clipboard from the previous steps – if not copy/paste it exactly as it appears on the options page.
+4. Click the ‘Create Client’ button, and you should now see the new client in the list.
+5. Click the downward facing caret on the right side to expose additional information about the client
+6. You should see something like this
+![](https://uiux.me/wp-content/uploads/2017/06/wordpress-install-ss4.png)
```
Comment (telaviv) [inline README.md]: this is a broken link

File: README.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+1. Install the plugin
+3. Activate the plugin and then click on ‘Patreon Settings’ to view the options page. It will look something like the below, copy the ‘Redirect URI‘ into the clip board and keep for the next step.
+![](https://c5.patreon.com/external/platform/wordpress-install-ss1.jpeg)
+
+### Creating an oAuth client on Patreon
+
+1. Ensure you are logged into Patreon, using your creator account. When it comes to testing this out you will want to have two Patreon accounts, one that acts as a patron and one being the content creator. The oAuth client needs to be created on your content creating account.
+2. Visit the [oAuth client page here](https://www.patreon.com/portal/registration/register-clients) and click "Create Client", fill out the form and add in an image URL for the icon that will appear to users when they are connecting to your website.
+3. Note the field for ‘Redirect URI‘, this should still be in your clipboard from the previous steps – if not copy/paste it exactly as it appears on the options page.
+4. Click the ‘Create Client’ button, and you should now see the new client in the list.
```
Comment (telaviv) [inline README.md]: here you do like weird unicode quotes, above you use double quotes. Be consistent and I might suggest using backticks for the cool `code` effect

File: README.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+1. Copy and paste the Client ID and Client Secret from this page into the matching fields on the ‘Patreon Settings’ page.
+2. Copy the Client ID from the top of this page and paste into the Creator ID field.
+3. Hit ‘Update Settings’
+4. Then go to wordpress ‘Settings’ -> ‘Permalinks’ and hit ‘Save’. This will ensure your rewrite rules are flushed.
+
+You should now be up and running with the Patreon WordPress plugin!
+
+## Protecting Posts and Pages
+
+When editing a post or page (or a custom post type) you will see a meta box in the right column titled ‘Patreon Level’. This box contains a text field that lets you specify a minimum contribution level in dollars. This could be $1 or $1.40 or even $10000. This is entirely determined by you. It defaults to $0 even if left empty.
```
Comment (telaviv) [inline README.md]: `When editing a post or page (or a custom post type)` -> When editing a post, page, or custom post typ

**Comment by codebard** [comment]
Closed since the changes are not relevant anymore.

**Pull request closed without merging**